data/json/vehicle_parts.json

```diff
@@ -5334,5 +5334,27 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "min": 1, "max": 2}
         ]
+    },
+    {
+        "type" : "vehicle_part",
+        "id" : "vehicle_scoop",
+        "name" : "scoop",
+        "symbol" : "R",
+        "color" : "light_gray",
+        "broken_symbol" : ";",
+        "broken_color" : "dark_gray",
+        "durability" : 100,
+        "epower" : -50,
+        "size":400,
```

Comment:
I actually do use both of the values. Hmm. It needs to be a cargo part, but it also needs the bonus to control the maximum size of the items included. 

I'll just derive the maximum item pickup size from the storage capacity. 

Thanks for bringing this to my attention, I was not aware of this issue.